googleapis/google-cloud-node
# migrate code from googleapis/nodejs-datastream

- feat: initial stub of library
- feat: initial library generation (#1)
- chore: release 0.2.0 (#3)
- chore(deps): update dependency sinon to v11 (#5)
- chore(deps): update dependency mocha to v9 (#4)
- build(node): update deps used during postprocessing (#1243) (#6)
- chore: fix the wrong post processor image (#9)
- chore(deps): update dependency @types/node to v16 (#8)
- chore(deps): update dependency sinon to v12 (#10)
- docs(samples): add example tags to generated samples (#11)
- build: add generated samples to .eslintignore (#12)
- docs(node): support "stable"/"preview" release level (#1312) (#15)
- chore: add api_shortname and library_type to repo metadata (#14)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#17)
- test(nodejs): remove 15 add 16 (#1322) (#19)
- build: update copyright year to 2022 (#21)
- chore: update v2.12.0 gapic-generator-typescript (#23)
- feat!: Adding public Datastream client based on v1 API version: https://cloud.google.com/datastream/docs (#26)
- chore(deps): update dependency sinon to v13 (#25)
- docs(nodejs): version support policy edits (#1346) (#29)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#30)
- chore(main): release 1.0.0 (#27)
- chore: update v2.14.2 gapic-generator-typescript (#34)
- feat: promote to stable (#35)
- chore(main): release 1.1.0 (#36)
- build(node): update client library version in samples metadata (#1356) (#43)
- chore(deps): update dependency sinon to v14 (#46)
- build!: update library to use Node 12 (#48)
- chore(main): release 2.0.0 (#49)
- chore(deps): update dependency jsdoc-region-tag to v2 (#52)
- chore(deps): update dependency jsdoc-fresh to v2 (#51)
- feat: support regapic LRO (#53)
- chore(deps): update dependency linkinator to v4 (#56)
- chore(main): release 2.1.0 (#54)
- chore: remove unused proto imports (#57)
- fix: better support for fallback mode (#58)
- fix: change import long to require (#59)
- fix: do not import the whole google-gax from proto JS (#1553) (#63)
- fix: allow passing gax instance to client constructor (#62)
- chore(main): release 2.1.1 (#60)
- fix: preserve default values in x-goog-request-params header (#64)
- test: use fully qualified request type name in tests (#67)
- fix(deps): use google-gax v3.5.2 (#71)
- chore(deps): update dependency @types/node to v18 (#69)
- chore(deps): update dependency jsdoc to v4 (#73)
- fix: regenerated protos JS and TS definitions (#74)
- chore(main): release 2.1.2 (#65)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
